crowdsourcing/serializers/requester.py

```diff
@@ -2,10 +2,26 @@
 
 from crowdsourcing import models
 from rest_framework import serializers
+import datetime
 
 class RequesterSerializer(serializers.ModelSerializer):
+    total_projects = serializers.SerializerMethodField()
+    current_projects = serializers.SerializerMethodField()
+    waiting_projects = serializers.SerializerMethodField()
+
```

Comment:
yeah, eventually worker would have to be updated to, let's not break the concept, it's ok to have a method in the module viewset which gets by requester or by worker